src/features/duckplayer/text.js

```diff
@@ -80,29 +80,15 @@ export function nl2br (text) {
  */
 
 /**
- *  @type {Record<import('../duck-player').UISettings['overlayCopy'], OverlayCopyTranslation>}
+ *  @type {Record<'default', OverlayCopyTranslation>}
```

Comment:
I didn’t want to roll back the experiment logic completely, as to leave the door open for future experiments.

Reply:
@shakyShane Plus, string ids such as `videoOverlayTitle2` are already being used on the mobile overlay, so I figured it best not to touch that part of the code just yet. But happy to refactor those names if you think it’s the right time to do that.